app/controllers/api/v1/admin/tenants_controller.rb

```diff
@@ -37,7 +37,7 @@ def index
 
         # POST /api/v1/admin/tenants
         def create
-          name = tenant_params[:name].downcase.gsub(/[-\s]/, '_')
+          name = tenant_params[:name]
```

Comment:
We do trust the admin to provide conform names?

Reply:
Yep - we need `-` in the names